### PR TITLE
basic instructions for profiling the ref server

### LINKED ISSUE #1192: document how to run development server with profiling
running the development server with profiling is tricky because flask gets in the way by starting a second process for the reloader functionality.  Please document the process:

`python -m cProfile -s cumulative -o profile.out ko-server/server_dev.py -H 100.70.26.73 --dont-use-reloader --config-file /srv/ga4gh/config.py`

### PR DESCRIPTION
**Pull request by kozbo**
This documents the lessons learned while we profiling the server code. Closes #1192

### CODE REVIEW

File: docs/ref_server_profiling.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+.. code-block:: bash
+
+    python -m cProfile -s cumulative -o profile.out ko-server/server_dev.py -H 100.70.26.73 --dont-use-reloader --config-file /srv/ga4gh/config.py
+
+******************************
+Evaluating the profile output:
+******************************
+
+If you don't use teh -o {output file} option then all output from the profiler
```
Comment (jeromekelleher) [inline docs/ref_server_profiling.rst]: s/teh/the

File: docs/ref_server_profiling.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+runs to stdout and will be sorted with the function that takes the most
+cumulative time at the top of the list. You can review the text output
+manually, but a beet method is to use the -o option and then review the
```
Comment (jeromekelleher) [inline docs/ref_server_profiling.rst]: a better?

**Comment by jeromekelleher** [comment]
Useful information, definitely something we want. Where does this show up in the HTML build? I think it should be linked into the docs tree somewhere, but I'm not sure where. I'm happy for it to be an orphan until we do a docs reorg though.

+1

**Comment by kozbo** [comment]
Thanks @jeromekelleher, this page is linked in from the development.rst page.

**Comment by kozbo** [comment]
@jeromekelleher Updates applied

**Comment by kozbo** [comment]
@jeromekelleher changes made

**Comment by jeromekelleher** [comment]
Merging; thanks @kozbo.

**Pull request merged**